### PR TITLE
8.0 template packages

### LINKED ISSUE #4328: Add aliases for common item templates
Consider adding the alternative short names to some item templates, they are often misspelled.
Support for several short names is already available.

- `gitignore` - `.gitignore`
- `globaljson` - `global.json`
- `editorconfig` - `.editorconfig`
- `nugetconfig` - `nuget.config`

### LINKED ISSUE #29059: Update template metadata for net8.0
### Describe the bug
Create new project with NET 8 SDK, net7.0 project is created instead

### To Reproduce
Install net8 SDK.
dotnet new console

### Exceptions (if any)
None

### PR DESCRIPTION
**Pull request by vlada-shubina**
Prepared 8.0 template packages

also fixes https://github.com/dotnet/templating/issues/4328
fixes https://github.com/dotnet/sdk/issues/29059

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Common.ProjectTemplates.8.0/content/ClassLibrary-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -47,9 +47,9 @@
       "datatype": "choice",
       "choices": [
         {
-          "choice": "net7.0",
-          "description": "Target net7.0",
-          "displayName": ".NET 7.0"
+          "choice": "net8.0",
+          "description": "Target net8.0",
+          "displayName": ".NET 8.0"
```
Comment (vlada-shubina) [inline template_feed/Microsoft.DotNet.Common.ProjectTemplates.8.0/content/ClassLibrary-CSharp/.template.config/template.json:52]: @baronfel  do we need to keep `standard` options?

Comment (vlada-shubina) [reply template_feed/Microsoft.DotNet.Common.ProjectTemplates.8.0/content/ClassLibrary-CSharp/.template.config/template.json:52]: Yes, they should be kept

**Comment by vlada-shubina** [comment]
Remaining tests won't pass until it is possible to build `net8.0` projects

**Review by JanKrivanek — APPROVED** [review]

**Comment by vlada-shubina** [comment]
Blocked by: https://github.com/dotnet/sdk/pull/28563

**Review by JanKrivanek — APPROVED** [review]
Looks good

**Pull request merged**